tests/wamr-test-suites/spec-test-script/runtest.py

```diff
@@ -1119,9 +1119,6 @@ def compile_wasm_to_aot(wasm_tempfile, aot_tempfile, runner, opts, r, output = '
         cmd.append("--enable-gc")
         cmd.append("--enable-tail-call")
 
-    if opts.memory64:
-        cmd.append("--enable-memory64")
```

Comment:
Seems we don't need to add `--enable-memory64` flag for wamrc, it can auto generate the correct aot file according to the memory definition in the wasm file?

Reply:
yes, that's why I removed this option.